homeassistant/components/pi_hole/const.py

```diff
@@ -20,7 +20,7 @@
 
 ATTR_BLOCKED_DOMAINS = "domains_blocked"
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
```

Comment:
This is useful for the status sensor, but for the others, it will increase network traffic unnecessarily (unless you what a nearly real-time view of Pi-hole stats in HA?)

I wonder if its better to just decrease the timeout for only the status sensor.

Reply:
Thanks for the feedback! I'll dig deeper and figure out what's the best way.